### DISCUSSION TITLE
lightspark Matrix server

### DISCUSSION THREAD

**Discussion by mrb0nk500** [lightspark/lightspark#1078] [Ideas]
I've thought about making a [Matrix](https://matrix.org) server for lightspark.

It's not only open, but federated as well, and seems like a better choice, compared to Discord.

Just wonder if you guys think it's a good idea.

**Comment by mrb0nk500** [comment]
Plus, if we want to, we can always add a Discord bridge, if that's necessary.

**Comment by eiglow** [comment]
Personally I'd be fine with this. I don't really use matrix much, I find the UX frustrating, but if you're willing to run the server, I think it could be good.

**Reply by mrb0nk500** [reply]
btw, the Matrix server is up, link is in the message below

**Comment by mrb0nk500** [comment]
Just made the Matrix server, and space.

https://matrix.to/#/#lightspark:b0nk.xyz

**Comment by mrb0nk500** [comment]
@dbluelle

**Comment by mrb0nk500** [comment]
I've currently only got a General room at the moment, but we can figure out stuff as we go.

**Comment by mrb0nk500** [comment]
I'll probably make a PR to add a badge for this.
